main.css

```diff
@@ -175,4 +175,24 @@ textarea {
     background-color: #D80;
     font-size: larger;
     color: white;
+
+}
+
+.line-numbers {
```

Comment:
@Alina-Ijaz , .line-number class is already at line 50. Will you remove this duplicate one?